src/components/Select.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+
+const Select = ({ disabled, ...props }) => (
+  <div
+    css={css`
+      height: 28px;
+      display: grid;
+      grid-template-areas: 'select';
+      align-items: center;
+      width: 100%;
+      min-width: 15ch;
+      border: 1px solid var(--color-neutrals-500);
+      border-radius: 0.25rem;
+      padding: 0.25rem 0.5rem;
+      cursor: pointer;
+      position: relative;
+
+      ${disabled &&
+      css`
+        cursor: not-allowed;
+        opacity: 0.5;
+
+        .light-mode & {
+          background-color: var(--color-neutrals-100);
+        }
+      `}
+
+      .dark-mode & {
+        border: 1px solid var(--color-dark-300);
+        background-color: var(--color-dark-300);
+      }
+
+      &::after {
+        content: '';
+        grid-area: select;
+        justify-self: end;
+        width: 0.5rem;
+        height: 0.25rem;
+        clip-path: polygon(100% 0%, 0 0%, 50% 100%);
+        background-color: var(--color-neutrals-700);
+
+        .dark-mode & {
+          background-color: var(--color-dark-700);
+        }
+      }
+    `}
+  >
+    <select
+      disabled={disabled}
+      css={css`
+        grid-area: select;
+        appearance: none;
+        background-color: transparent;
+        border: none;
+        padding: 0 1em 0 0;
+        margin: 0;
+        width: 100%;
+        font-family: inherit;
+        font-size: 14px;
+        cursor: inherit;
+        line-height: inherit;
+        outline: none;
```

Comment:
As much as I like this style, it's an [accessibility issue](http://www.outlinenone.com/) to remove `outline`:
```suggestion
```